content/sensu-core/1.4/guides/monitor-the-monitor.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Monitoring Sensu with Sensu"
+description: "Strategies and best practices for monitoring Sensu with Sensu"
+version: "1.4"
+weight: 10
+previous: ../securing-redis
+menu:
+  sensu-core-1.4:
+    parent: guides
+---
+
+In this guide, we'll walk you through the best practices and strategies for monitoring Sensu with Sensu. By the end of the guide, you should have a thorough understanding of what is required to ensure your Sensu components are properly monitored, including:
+
+* How to monitor your Sensu server and Sensu API instance(s)
+* How to monitor your Uchiwa Dashboard instance(s)
+* How to monitor your RabbitMQ instance(s)
+* How to monitor your Redis instance(s)
+
+_NOTE: In order to completely monitor a Sensu stack (Sensu services, Redis, RabbitMQ), you will need to have at least one other independent Sensu stack to do so. This is due to a single Sensu stack can not monitor itself completely, as if some components are down, Sensu will not be able to alert on itself properly_
```

Comment:
This content is essential for the guide, so I'd suggest removing the NOTE formatting

Reply:
Done. I wasn't entirely sure where to put it, as I wanted everyone reading the guide to know that it is a requirement for some of the services. I do try to call it out at the beginning of the checks that require it, so maybe that is enough.